server

```diff
@@ -114,14 +52,23 @@ check_product() {
   esac
 }
 
-is_rc_version() {
-    if [[ $SCYLLA_VERSION == *.rc* ]]; then
-      # replace the 2nd '.' with '~' for RC versions, ex. 4.6.rc0 -> 4.6~rc0
-      SCYLLA_VERSION=$(echo $SCYLLA_VERSION | sed 's/\(.*\)\.)*/\1~/')
-    elif [[ $SCYLLA_VERSION == *-rc* ]]; then
-      # replace '-' with '~' for RC versions, ex. 5.1.0-rc1 -> 5.1.0~rc1
-      SCYLLA_VERSION=$(echo $SCYLLA_VERSION | sed 's/-/~/')
+check_scylla_version() {
+  if [[ ! -z "$SCYLLA_VERSION" ]] && [[ $SCYLLA_VERSION != *"nightly"* ]]; then
+    if [[ "$SCYLLA_VERSION" != @(2022.[1-2]*|202[3-9].*|5.[2,4-9]*|[6-9].*) ]]; then
+      echo "The specified scylla-version '$SCYLLA_VERSION' has reached End of Life (EOL) and is no longer supported. For supported ScyllaDB versions please refer to https://docs.scylladb.com/getting-started/os-support"
```

Comment:
The $SCYLLA_VERSION will not interpolate because it wraps with a single-quotes. you can remove the quotes

Reply:
According to the verificatin it will looks like that:
`The specified scylla-version '2021.1.0' has reached End of Life (EOL) and is no longer supported.  For supported ScyllaDB...`